datasource/models/wikidata.py

```diff
@@ -30,7 +30,7 @@ def load_and_create(cls, load_from_api=False):
             print("Loading Wikidata data from local copy...")
             df = pd.read_csv("./datasource/local/wikidata/wikidata.csv")
         else:
-            print("Loading Wikidata data from API...")
+            print("Loading Wikidata data from api...")
```

Comment:
Did you intend to replace 'API' with 'api' in all these `datasource/models/..` files or was this an accident?

Reply:
Previously I had created folder named API, but later I found that all other folders name is in lower case so changed folder name from API to api. I have not made any changes in this file (datasource/models/wikidata.py)

Reply:
In that case, could you remove all these from the PR. There are several files (I think 9) where the only difference is that 'API' is changed to 'api' in various comments or message strings.